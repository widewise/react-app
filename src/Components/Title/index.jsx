import React from 'react';
import styled from 'styled-components';

const TitleString = styled.h1`
    background-color: #424242;
    color: #F65261;
    margin: 0;
    flex: 1;
`;
const NetflixTitleString = styled.strong`
    font-size: 110%;
`;

export default function Title() {
  return (
    <>
      <TitleString>
        <NetflixTitleString>netflix</NetflixTitleString>
        roulette
      </TitleString>
    </>
  );
}
