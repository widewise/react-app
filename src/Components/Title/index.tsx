import React, { FunctionComponent } from 'react';
import styled from 'styled-components';

const TitleString = styled.h1`
    background-color: #424242;
    color: #F65261;
    margin: 0;
    flex: 1;
`;
const NetflixTitleString = styled.strong`
    font-size: 110%;
`;

const Title: FunctionComponent = () => (
  <>
    <TitleString>
      <NetflixTitleString>netflix</NetflixTitleString>
      roulette
    </TitleString>
  </>
);

export default Title;
