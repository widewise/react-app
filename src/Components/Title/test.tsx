import React from 'react';
import { render } from '@testing-library/react';
import Title from './index';

describe('Title', () => {
  it('renders Title snapshot', () => {
    const { asFragment } = render(<Title />);
    expect(asFragment()).toMatchSnapshot();
  });
});
