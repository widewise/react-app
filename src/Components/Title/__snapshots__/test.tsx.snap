// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders Title snapshot 1`] = `
<DocumentFragment>
  <h1
    class="sc-bdfBwQ htrgWn"
  >
    <strong
      class="sc-gsTCUz iznvlr"
    >
      netflix
    </strong>
    roulette
  </h1>
</DocumentFragment>
`;
